notebooks/gpu-footprint.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        pipe = pipeline(
</pre><p> </p><p>For the purposes of this notebook it should be enough to do the model load (<code>model = auto_class...</code>), probably no need to create the pipeline here, as it is not used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>